### DISCUSSION TITLE
Azure Easy Authentication

### DISCUSSION THREAD

**Discussion by michielwillekens** [AgileTek/django-azure-auth#47] [Ideas]
Hi, have you considered support Azure Easy Authentication? I am using Azure Container Apps and restrict all access to the container app by azure authentication. However, this is not picked up by this plugin. Supporting this would help me a lot, have you considered this? I could help with the implementation.

**Comment by regoawt** [comment]
I'm not too familiar with Azure Easy Authentication so I might be misunderstanding its use case, but a cursory look at https://learn.microsoft.com/en-us/azure/app-service/overview-authentication-authorization suggests it would be replacement/alternative to django-azure-auth, rather than something DAA would hook into? 

Happy to be convinced otherwise.
